geoh5py/shared/concatenation/object.py

```diff
@@ -38,14 +36,39 @@ class ConcatenatedObject(Concatenated, ObjectBase, ABC):
     _property_groups: list | None = None
 
     def __init__(self, entity_type, **kwargs):
-        if kwargs.get("parent") is None or not is_concatenator(kwargs.get("parent")):
+        if kwargs.get("parent") is None:
             raise UserWarning(
                 "Creating a concatenated object must have a parent "
                 "of type Concatenator."
             )
 
         super().__init__(entity_type, **kwargs)
 
+    def add_children(
+        self,
+        children,
+    ):
+        """
+        :param children: Add a list of entities as
+            :obj:`~geoh5py.shared.entity.Entity.children`
+        """
+        property_groups = self._property_groups or []
+
+        for child in children:
+            if child not in self._children and isinstance(
```

Comment:
a property group is both in children and property_groups?

Reply:
Correct, we did that change together last year, so we can access prop_groups from the top

Reply:
You're right